README.md

```diff
@@ -11,7 +11,7 @@ Despite reporting on the same topics, different media organisations can have wil
 
 ## About the Project
 
-This project extracts UK news articles from different RSS feeds and finds related articles from other news feeds as well as comments about that story/topic on Reddit. Sentiment analysis is then applied to these news articles and social media comments and scores them depending on how positive/negative they are (-1 being the most negative, 1 being the most positive). The articles then have their topic keywords extracted using ChatGPT, this allows for articles/social media posts talking about the same news story/topic can be linked together. All of this information is then loaded onto an RDS which is running PostgreSQL. From the RDS, we visualise the wide variety of information we have gathered via a Tableau dashboard and also send out an email report with a PDF attached showing the most popular stories of the day amongst other key insights.  
+This project extracts UK news articles from different RSS feeds and finds related articles from other news feeds as well as comments about that story/topic on Reddit. Sentiment analysis is then applied to these news articles and social media comments and scores them depending on how positive/negative they are (-1 being the most negative, 1 being the most positive). The articles then have their topic keywords extracted using ChatGPT, this allows for articles/social media posts talking about the same news story/topic can be linked together. All of this information is then loaded onto an RDS which is running PostgreSQL. From the RDS, we visualise the wide variety of information we have gathered via a Tableau dashboard and also send out an email report with a PDF attached showing the most popular stories of the day amongst other key insights.
```

Comment:
As we are only using BBC and DailyMail, it might be worth mentioning these two sources here.

Reply:
The phrase "this allows for articles/social media posts talking about the same news story/topic can be linked together." may need to be adjusted as it is not grammatically correct